packages/thirdweb/src/wallets/in-app/web/lib/web-connector.ts

```diff
@@ -111,6 +130,37 @@ export class InAppWebConnector implements InAppConnector {
     });
   }
 
+  async initializeWallet(authToken?: string) {
+    const storedAuthToken = await getAuthToken(this.client, this.ecosystem);
+    if (!authToken && storedAuthToken === null) {
+        throw new Error("No auth token provided and no stored auth token found to initialize the wallet");
+    }
+
+    const user = await getEnclaveUserStatus({ authToken: authToken || storedAuthToken as string, client: this.client, ecosystem: this.ecosystem });
+    if (user.wallets[0].type === "enclave") {
```

Comment:
user can be undefined? / wallets can be empty? make sure to guard against that

Reply:
The backend _should_ always send exactly one wallet but I added a check in `getEnclaveUserStatus` to throw if that doesn't happen for some reason